src/test/kotlin/com/atlassian/performance/tools/jvmtasks/api/TaskTimerTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.atlassian.performance.tools.jvmtasks.api
+
+import com.atlassian.performance.tools.jvmtasks.api.TaskTimer.time
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import java.time.Duration
+
+class TaskTimerTest {
+    private val handler = DummyEventHandler()
+
+    @Before
+    fun setUp() {
+        TaskTimer.registerTaskStartedHandler(handler)
+        TaskTimer.registerTaskSucceededHandler(handler)
+        TaskTimer.registerTaskFailedHandler(handler)
```

Comment:
Is it fine that we don't deregister this anymore? It leaves handlers inside `TaskTimer` if we run the test multiple times in the same VM.

Reply:
Each test has it's own `path: List<String>`, so they'll be separated.
However, if we register (sub) the same handler multiple times, then we'll get duplicated pubs.